src/tests/ftest/util/daos_utils.py

```diff
@@ -89,6 +89,10 @@ def container_create(self, pool, sys_name=None, cont=None,
             CommandFailure: if the daos container create command fails.
 
         """
+        if properties is None:
+            properties.append('rf_lvl:1')
+        else:
+            properties.append(',rf_lvl:1')
```

Comment:
Not sure if this PR is ready for review yet, but I don't think we should be setting this at this level. For example, what if a test wants to create a container with `rf_lvl:2`? The properties would become `rf_lvl:2,rf_lvl:1`. Or what if a test specifically wants to create a container WITHOUT specifying `rf_lvl`?

Reply:
This PR is ready for review if can pass CI.
the story is originally the rf_lvl is not supported and all set as RANK by default, this patch change the default as NODE (2). To make it don't break current tests, it just create container with rf_lvl:1, because a few tests may break if create container with default 2 now.
I think this may need help from test engineer to add support of other rf_lvl later to test script if needed later.
The previous patch added a C test case that tested rf_lvl:2, this patch just change default value as 2 but change all other tests use rf_lvl:1 same as before.